src/gp-model/helpers/get_data.py

```diff
@@ -0,0 +1,170 @@
+#importing packages
+from pymongo import MongoClient
+from datetime import datetime, timedelta
+import requests
+import os
+from dotenv import load_dotenv
+load_dotenv()
+
+EVENTS_URL = os.getenv('EVENTS_URL')
+MONGO_URI = os.getenv('MONGO_GCE_URI')
+client = MongoClient(MONGO_URI)
+
+def connect_db(owner):
+    """Connects to database
+
+    Parameters
+    ----------
+    owner : str
+        The database owner
+
+    Returns
+    -------
+    db : Database
+        a MongoDB connection
+    """
+    db_name = f'airqo_netmanager_{owner}'
+    db = client[db_name]
+    return db
+
+def get_device_details(device_id, owner):
+    """Returns a device's details given the ID
+
+    Parameters
+    ----------
+    device_id : str
```

Comment:
I guess this is suppose to be `device_number` ??

Reply:
@danielogen, it is a device identifier. For example KCCA devices don't have numbers